### ISSUE TITLE
"Packaging" conflicts with "robocorp-flet" and "rpaframework-core"

### ISSUE THREAD

**Issue by axiom41** [robocorp/rpaframework#1183]
Hello.
After updating rpaframework-core to 11.3.2, the "packaging" package is upgraded to 24.0 and the following compatibility error shows up:

`robocorp-flet 0.4.2.1 requires packaging<24.0,>=23.0, but you have packaging 24.0 which is incompatible.`

But reinstalling robocorp-flet, "packaging" package is downgraded to 23.2, so i get instead:

`rpaframework-core 11.3.2 requires packaging<25.0,>=24.0, but you have packaging 23.2 which is incompatible.`

Probably your recently released robocorp-flet needs an update too, as well as all of its other depending packages...

Many thanks in advance for your support.

**Comment by raivolink** [comment]
Hello, what is the content of your `conda.yaml`? You should use the `rpaframework-assistant` package with the latest version. The `robocorp-flet` is a fork of the original package as `flet` removed some of the packages. [Release notes](https://rpaframework.org/releasenotes.html#id6)

**Comment by axiom41** [comment]
Hello.
I could not find any "conda.yaml" file in my system (win11 x64, Python 3.11.x) and rpaframework-assistant is 3.0.4. I tried to reinstall it and it restored rpaframework-core 11.3.1, which is compatible with robocorp-flet 0.4.2.1.

So currently compatibility issues were resolved by downgrading rpaframework-core to 11.3.1, but then why did you publish rpaframework-core 11.3.2 only if other packages were not updated?

**Comment by axiom41** [comment]
Hello.
I could not find any "conda.yaml" file in my system (win11 x64, Python 3.11.x) and rpaframework-assistant is 3.0.4. I tried to reinstall it and it restored rpaframework-core 11.3.1, which is compatible with robocorp-flet 0.4.2.1.

So currently compatibility issues were resolved by downgrading rpaframework-core to 11.3.1, but then why did you publish rpaframework-core 11.3.2 only if other packages were not updated?
